scaffold_test.go

```diff
@@ -17,13 +19,34 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+var (
+	dbURL  string
+	ssoURL string
+	bToken string
+)
+
 var insecureClient = &http.Client{
 	Transport: &http.Transport{
 		TLSClientConfig: &tls.Config{
 			InsecureSkipVerify: true,
 		},
 	},
 }
+var _ = BeforeSuite(func() {
+	ssoURL = os.Getenv("TEST_SSO_URL")
+	bToken = os.Getenv("BEARER_JWT_TOKEN")
+	if ssoURL == "" || bToken == "" {
+		fmt.Println("Tests aborted: TEST_SSO_URL/BEARER_JWT_TOKEN not set\n")
+		fmt.Println("Example:")
+		fmt.Println("TEST_SSO_URL=https://login.e2e.apigee.net/token_key")
+		fmt.Println("BEARER_JWT_TOKEN=eyJhbGciOiJSUzI1NiJ9.eyJqdGkiOiIwMDgwNWNlYi0yNzI5LTQ2OTgtYWNiMy1jNTRkZmIzMWM4MjEiLCJzdWIiO\n")
+		fmt.Println("NOTE:-")
+		fmt.Println("BEARER_JWT_TOKEN can be gotten by `get_token -u sramamoorthy+edgetest@apigee.com:Test12345`")
```

Comment:
Hmm. I wanted folks to use this account/password on purpose. If you feel this is not needed, I can just add dummy user/password.